doc/introduction.rst

```diff
@@ -14,4 +14,43 @@ and local PC.
 
 .. note:: Because of various technical issues, desktop sync clients older than 
    1.7 will not allowed to connect and sync with the ownCloud 8.1 server. It is 
-   highly recommended to keep your client updated.
\ No newline at end of file
+   highly recommended to keep your client updated.
+   
+Improvements and New Features
+-----------------------------
+
+The 2.0 release of the ownCloud desktop sync client has many new features and 
+improvements.
+
+  * Multi-account support
+  * Many UI improvements
+  * Accessibility improvements (high contrast schemes)
+  * Automatic Bandwidth Throttling
+  * Don't show redundant directory entries in activity log
+  * Remove deleted accounts properly from toolbar
+  * File manager integration: Show hidden files as ignored
+  * Show wizard when last account was deleted
+  * Do not sync down new big folders from server without user's consent
+  * Integrate Selective Sync into the default UI
+  * More reliable reconnect after timeout
+  * Use SI units for the file sizes
```

Comment:
What to say here? In 2.0.2 it goes back to showing units as KB, MB, GB but actually calculating underneath in 1024 multiples.